pillar/top.sls

```diff
@@ -156,8 +156,8 @@ base:
   'G@roles:devstack and P@environment:dev':
     - match: compound
     - devstack
-    - consul_devstack
-    - mysql_devstack
-    - rabbitmq_devstack
-    - elasticsearch_devstack
-    - mongodb_devstack
+    - consul.devstack
+    - db.mongodb_devstack
+    - db.mysql_devstack
```

Comment:
The references for mongo and mysql need to be updated to point to their new paths.